README.md

```diff
@@ -4,22 +4,94 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to an OpenShift cluster
```

Comment:
I wonder if it's necessary to point out that it could be (local or remote) to indicate that something like CRC/OpenShift Local would be applicable as mentioned below?